Content/RecipeChanges/GalacticaSingularity.cs

```diff
@@ -0,0 +1,29 @@
+using Terraria;
+using Terraria.ID;
+using Terraria.ModLoader;
+using ThoriumMod.Items.BardItems;
+using ThoriumMod.Items.HealerItems;
+using ThoriumMod.Items.ThrownItems;
+using CalamityMod.Items.Materials;
+
+namespace TheBereftSouls.Content.RecipeChanges;
+
+[ExtendsFromMod("CalamityMod")]

```

Comment:
you need to add thorium to the extends from 
should be like this
```suggestion
[ExtendsFromMod(["CalamityMod","ThoriumMod"])]

```

Reply:
Is there a reason to? The code works, seems like needless extra code for no benefit

Reply:
It shouldn't work unless you have thorium set as a strong dependancy. as is currently dictated by the build.txt thorium should be a weak refrence in this case if the mod can't find the thorium mod it will fail, this is not desired as the thorium mod is not a required dependancy.

Reply:
Ok, I understand that, but there's little point playing Bereft Souls without Thorium, so it doesn't make that much sense, but sure, I will add it anyway, I have a lot of appends to make, and I'm about to post my update commit soon